mmv1/products/artifactregistry/product.yaml

```diff
@@ -17,9 +17,12 @@ display_name: Artifact Registry
 scopes:
   - https://www.googleapis.com/auth/cloud-platform
 versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://artifactregistry.googleapis.com/v1/
```

Comment:
What is the reason to use v1 endpoint for beta?

Reply:
The feature is released to Public Preview but not GA. We are not adding new features to the v1beta2 endpoint.

Reply:
Yes.